### ISSUE TITLE
Do I have to authorize my store again for adding another marketplace?

### ISSUE THREAD

**Issue by Javadebi** [amzn/selling-partner-api-docs#2273]
Hello, I have already authorized my USA marketplace for my store, do I need to recall authorization for Canada marketplace with Canada celler central uri?

**Comment by stefnats** [comment]
Nope, granting access (= receiving the refresh token) is always for a complete region.
